.github/ISSUE_TEMPLATE/release-template.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Release template
+about: Internally used for new releases
+title: Release x.y.z
+labels: 'Type: Maintenance'
+assignees: jblz, GaryJones, mikeyarce, hbbtstar
```

Comment:
```suggestion
```

Bummer that assignees can be a group.  I'd probably leave it out so we don't have to change this to add / rm people